### ISSUE TITLE
Import Wegue as a module?

### ISSUE THREAD

**Issue by ntma** [meggsimum/wegue#232]
Hi Wegue team,

I am interested in using this package as a module in a [NuxtJs](https://nuxtjs.org/) app. To do so, I need to create a plugin that imports the required components/configs from Wegue... but I am currently struggling with:
* which components should I import;
* how to avoid `$appConfig` since in my case I want a configuration per instance.

**Comment by chrismayer** [comment]
Hi @ntma, thanks for your interest in Wegue. I fear at the moment it is not possible, what you are planning to do. There is no importable component package yet. We plan to separate the components from the template application in a mid-term change set. Hereby we want to decouple the components from the application configuration. So plans are there, but not implemented yet.

**Comment by ntma** [comment]
Ah, awesome! I'll look forward to it!

In the meanwhile, is there a roadmap settled for the development of Wegue that can be shared?

**Comment by chrismayer** [comment]
No, there is no official roadmap so far. But we should do that, to get things more transparent. The [issues](https://github.com/meggsimum/wegue/issues) give us some kind of backlog and currently the mid- and long-term things are more ideas evolving in talks between the developers.
